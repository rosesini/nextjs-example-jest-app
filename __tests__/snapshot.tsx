import { render } from '@testing-library/react'
import Home from '@/pages/index'

jest.mock('../lib/posts', () => {
  return {
    getSortedPostData: () => []
  }
})

it('renders homepage unchanged', () => {
  const { container } = render(<Home postsData={[]} />)
  expect(container).toMatchSnapshot()
})
