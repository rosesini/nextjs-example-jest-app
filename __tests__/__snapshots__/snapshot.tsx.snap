// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <header
      class="header"
    >
      <span
        style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
      >
        <span
          style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
        >
          <img
            alt=""
            aria-hidden="true"
            src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27108%27%20height=%27108%27/%3e"
            style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
          />
        </span>
        <img
          alt="RagedMist"
          class="borderCircle"
          data-nimg="intrinsic"
          decoding="async"
          src="/_next/image?url=%2Fimages%2Fprofile.jpg&w=256&q=75"
          srcset="/_next/image?url=%2Fimages%2Fprofile.jpg&w=128&q=75 1x, /_next/image?url=%2Fimages%2Fprofile.jpg&w=256&q=75 2x"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
      </span>
      <h1
        class="heading2Xl"
        data-testid="home__profile-name"
      >
        RagedMist
      </h1>
    </header>
    <main>
      <section
        class="headingMd padding1px"
      >
        <h2
          class="headingLg"
        >
          Blog
        </h2>
        <ul
          class="list"
        />
      </section>
    </main>
  </div>
</div>
`;
